docs/source/package_reference/stable_diffusion_pipeline.mdx

```diff
@@ -1,85 +0,0 @@
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
--->
-
-# GaudiStableDiffusionPipeline
-
-The `GaudiStableDiffusionPipeline` class enables to perform text-to-image generation on HPUs.
-It inherits from the `GaudiDiffusionPipeline` class that is the parent to any kind of diffuser pipeline.
-
-To get the most out of it, it should be associated with a scheduler that is optimized for HPUs like `GaudiDDIMScheduler`.
-
-
-## GaudiStableDiffusionPipeline
-
-[[autodoc]] diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.GaudiStableDiffusionPipeline
-    - __call__
-
-
-## GaudiDiffusionPipeline
-
-[[autodoc]] diffusers.pipelines.pipeline_utils.GaudiDiffusionPipeline
-    - all
-
-
-## GaudiDDIMScheduler
-
-[[autodoc]] diffusers.schedulers.scheduling_ddim.GaudiDDIMScheduler
-    - all
-
-
-# GaudiStableDiffusionXLPipeline
-
-The `GaudiStableDiffusionXLPipeline` class enables to perform text-to-image generation on HPUs using SDXL models.
-It inherits from the `GaudiDiffusionPipeline` class that is the parent to any kind of diffuser pipeline.
-
-To get the most out of it, it should be associated with a scheduler that is optimized for HPUs like `GaudiDDIMScheduler`.
-Recommended schedulers are `GaudiEulerDiscreteScheduler` for SDXL base and `GaudiEulerAncestralDiscreteScheduler` for SDXL turbo.
-
-
-## GaudiStableDiffusionXLPipeline
-
-[[autodoc]] diffusers.pipelines.stable_diffusion_xl.pipeline_stable_diffusion_xl.GaudiStableDiffusionXLPipeline
-    - __call__
-
-
-## GaudiEulerDiscreteScheduler
-
-[[autodoc]] diffusers.schedulers.scheduling_euler_discrete.GaudiEulerDiscreteScheduler
-    - all
-
-
-## GaudiEulerAncestralDiscreteScheduler
-
-[[autodoc]] diffusers.schedulers.scheduling_euler_ancestral_discrete.GaudiEulerAncestralDiscreteScheduler
-    - all
-
-
-# GaudiStableDiffusionUpscalePipeline
-
-The `GaudiStableDiffusionUpscalePipeline` is used to enhance the resolution of input images by a factor of 4 on HPUs.
-It inherits from the `GaudiDiffusionPipeline` class that is the parent to any kind of diffuser pipeline.
-
-
-[[autodoc]] diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_upscale.GaudiStableDiffusionUpscalePipeline
-    - __call__
-
-
-# GaudiDDPMPipeline
-
-The `GaudiDDPMPipeline` is to enable unconditional image generations on HPUs. It has similar APIs as the regular `DiffusionPipeline`.
-It shares a common parent class, `GaudiDiffusionPipeline`, with other existing Gaudi pipelines. It now supports both DDPM and DDIM scheduler. 
-It is recommended to use the optimized scheduler, `GaudiDDIMScheduler`, to obtain the best performance and image outputs.
-
```

Comment:
why remove this?

Reply:
Greg suggested to upgrade to SDXL as SD is kind of old model now.  We can keep too, it may be a good intro diffuser model.

Reply:
@kplau1128 can you comment here?

Reply:
Greq was request and suggested consolidated SD into one tutorials/stable_diffusion.mdx, and remove others